lib/modules/onc_program/bulk_data_export_sequence.rb

```diff
@@ -276,6 +276,29 @@ def assert_requires_access_token(status_response = @status_response)
         assert_requires_access_token
       end
 
+      test :bulk_data_delete_test do
+        metadata do
+          id '10'
+          name 'Bulk Data support DELETE request'
+          link 'http://hl7.org/fhir/uv/bulkdata/export/index.html#bulk-data-delete-request'
+          description %(
+            Bulk Data Server SHALL support client DELETE request
+         )
+        end
+
+        reply = export_kick_off(endpoint, resource_id)
+
+        assert_response_accepted(reply)
+
+        content_location = reply.headers[:content_location]
+
+        assert content_location.present?, 'Export response header did not include "Content-Location"'
+
+        reply = LoggedRestClient.delete(content_location, build_header({ accept: 'application/json' }))
+
+        assert_response_accepted(reply)
```

Comment:
We check that the server accepts the deletion request, but is it possible to also check that the request actually was deleted?

Reply:
The problem is that IG does not specify that server shall stop the export process immediately or stop the process at all.